### DISCUSSION TITLE
Keilsilo mit Wartebereich auf beiden Seiten

### DISCUSSION THREAD

**Discussion by DanielSan85** [Courseplay/Courseplay_FS22#3074] [Q&A] [answered]
Wie im Bild unten zu sehen ist hat mein Keilsilo leider auch an der Rückseite einen Wartebereich verpasst bekommen.
Das führt dazu, dass jeder Abfahrer der auf der Straße dort passieren will meinen CP Helfer auf seine Warteposition schickt.
Der Abfahrer auf den in diesem Bild gewartet wird sieht man oben links hinter den Bäumen mit der gelben Plane gerade losfahren.

Gibt es eine Möglichkeit, den Bereich an der Rückwand des Silos nicht mit so einem Wartebereich zu versehen?

![grafik](https://github.com/Courseplay/Courseplay_FS22/assets/18481866/2e5f506a-9def-42e4-a058-3abedad99867)

**Comment by Tensuko** [comment, marked as answer]
Speicher mal dein spiel und starte neu und guck ob das dann noch da ist (neueste 7.4.0.2 vorrausgesetzt).
Wir haben einen Plan wie man das besser machen kann (wenn Zeit dafür da ist), aktuell geht es leider nicht anders.

**Comment by DanielSan85** [comment]
Vielen Dank; der Neustart hat das Problem gelöst.
